packages/svelte/src/routes/+page.svelte

```diff
@@ -1,3 +1,5 @@
+<svelte:options runes={false} />
```

Comment:
Maybe we don't need to disable runes here. I think we only need it in the Dotlottie.svelte component. For the DotLottie Svelte consumers, they don't have to worry about disabling the runes, as far as I understand

Reply:
Yea you're right, we don't need it here.
I added that just in case (as a part of testing), sure I will remove it right away.

Reply:
Resolved on https://github.com/LottieFiles/dotlottie-web/pull/217/commits/1d50b191beb7f2c03b9837bb033126d26d8708bd